internal/conf/configuration.go

```diff
@@ -111,6 +111,10 @@ func (a *APIConfiguration) Validate() error {
 	return nil
 }
 
+type SessionsConfiguration struct {
+	Timebox time.Duration `json:"timebox"`
```

Comment:
personal preference to make this a pointer so it defaults to `nil` if not set instead of time.Duration(0)

Reply:
Should we forbid negative durations as well?